src/components/Review/ReviewSection.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Button, Grid, Header, Segment } from 'semantic-ui-react'
+import { SummaryViewWrapper } from '../../formElementPlugins'
+import { TemplateElementCategory } from '../../utils/generated/graphql'
+import { SectionElementStates } from '../../utils/types'
+
+interface ReviewSectionProps {
+  reviewSection: SectionElementStates
+  canEdit: boolean
+}
+
+const ReviewSection: React.FC<ReviewSectionProps> = ({ reviewSection, canEdit }) => {
+  const { section, pages } = reviewSection
+  return (
+    <Segment inverted style={{ backgroundColor: 'WhiteSmoke', margin: '15px 50px 0px' }}>
+      <Header as="h2" content={`${section.title}`} style={{ color: 'Grey' }} />
+      {Object.entries(pages).map(([pageName, elements]) => (
+        <Segment basic>
+          <Header as="h3" style={{ color: 'DarkGrey' }}>
+            {pageName}
+          </Header>
+          {elements.map(({ element, response, review }) => {
+            const { category, isVisible, isEditable } = element
+            return isVisible ? (
+              <Segment key={`ReviewSection_${element.code}`} style={{ margin: 10 }}>
+                <Grid columns={2} verticalAlign="middle">
+                  <Grid.Row>
+                    <Grid.Column width={13}>
+                      {/* // TODO: Replace with SummaryViewWrapper */}
+                      <Header>{element.title}</Header>
+                      {response && <Segment basic>{response?.text}</Segment>}
+                    </Grid.Column>
+                    <Grid.Column width={3}>
+                      {category === TemplateElementCategory.Question &&
+                        isVisible &&
+                        isEditable &&
```

Comment:
Can probably allow reviews/comments on non-editable responses, right? For example, if a response was auto-generated based on other responses, we'd still want to be able to approve/comment, do you think?

Reply:
Ah, yea. Good point! I'll change this now.